### DISCUSSION TITLE
Feature Request: "View as" Mode on Github Profile

### DISCUSSION THREAD

**Discussion by diamant3** [community/community#39565] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Profile

### Body

I want this feature to allow me to see what others see in my GitHub profile.

**Comment by TheKodeToad** [comment]
![image](https://user-images.githubusercontent.com/57493648/203754122-3fd7068d-4d09-43bb-a7ed-008fe9474411.png)
You can already *kind of* do this if you're profile is private.

**Reply by diamant3** [reply]
Thank you for the answer but i mean of *view as* is for public view of our GitHub profile, it's like in Facebook that has *view as* feature.

**Reply by TheKodeToad** [reply]
It was intended more as a comment. I don't think you can do this on GH discussions though.

**Comment by airtower-luna** [comment, marked as answer]
Most browsers have a "private browsing" mode or similar. If you use that to look at your profile you should see it as if you aren't logged in.
